src/App.tsx

```diff
@@ -292,6 +292,20 @@ export default function App() {
         return;
       }
 
+      if (tabs?.length) {
+        tabs.forEach((_tab, i) => {
+          // cmd+2, etc.
+          if (event.metaKey && event.key === `${i + 1}`) {
+            event.preventDefault();
+            setTabIndex(i);
+            ipcRenderer.send(TAB_CHANGED, {
+              tab: tabs[i],
+              input: inputValue,
+            });
+          }
+        });
+      }
```

Comment:
pressing cmd + `number` switches to appropriate tab index. definitely up for discussion :)